ghga_service_chassis_lib/api.py

```diff
@@ -36,9 +37,25 @@ class ApiConfigBase(BaseSettings):
     log_level: LogLevel = "info"
     auto_reload: bool = False
     workers: int = 1
+    cors_allowed_origins: List[str] = []
+    cors_allow_credentials: bool = True
+    cors_allowed_methods: List[str] = ["*"]
+    cors_allowed_headers: List[str] = ["*"]
```

Comment:
I feel a bit uncomfortable about the FastAPI defaults being overridden here, especially as they are being relaxed. As a developer I might rely on them to be set to the FastAPI defaults unless overwritten in the config.

Reply:
I would say that every of our developers has to know exactly what the shared library is doing. Here I thought is a good place to define our own defaults, it's not a general-purpose library for all FastAPI developers. Don't you agree?

Reply:
I do, yet I would not expect that to be always the case. It puts additional pressure on the documentation. If I were a new developer with a FastAPI background I might for example assume that specifying `cors_allowed_origins` in the config will not also implicitly enable access to credentials. I would be more conservative, especially in a security related context, just my two cents :-)

Reply:
I would really consider it mandatory that every developer knows this, the shared library is not intended to hide complexity but only to guarantee consistency across all services. Every developer has to perfectly know what the ApiConfigBase contains.

But in this specific case I don't have a problem to consider our defaults for CORS methods/headers to be the one from Starlette.

Will change it

Reply:
How about this?

Reply:
Sorry, my time is again wrong in the container, the commit shows up at the top.

Reply:
Looks good to me!